frontend/src/pages/modelServing/screens/projects/utils.ts

```diff
@@ -645,7 +650,6 @@ export const createNIMPVC = (
     {
       dryRun,
     },
-    true,
```

Comment:
Why did you remove this? We still don't want those PVCs shown -- at least I don't think so... having them in the project's cluster storage seems to have little value 🤔 unless we build in a whole system to manage these I can't imagine this working out.

Reply:
![image](https://github.com/user-attachments/assets/2d753591-38eb-4a25-9223-f3f85d2f8c54)

This is what not providing this value gets you...